cmd/sonobuoy/app/gen.go

```diff
@@ -128,14 +133,16 @@ func (g *genFlags) Config() (*client.GenConfig, error) {
 
 	return &client.GenConfig{
 		E2EConfig:            e2ecfg,
-		Config:               g.getConfig(),
+		Config:               g.resolveConfig(),
 		Image:                g.sonobuoyImage,
 		Namespace:            g.namespace,
 		EnableRBAC:           rbacEnabled,
 		ImagePullPolicy:      g.imagePullPolicy.String(),
 		KubeConformanceImage: image,
 		SSHKeyPath:           g.sshKeyPath,
 		SSHUser:              g.sshUser,
+		DynamicPlugins:       g.plugins.dynamicPlugins,
+		StaticPlugins:        g.plugins.staticPlugins,
```

Comment:
These were put into the `GenConfig` and not the `Config` very consciously.  In the `Config` we used to give the list of plugins to run, but not the actual config. The static plugins in particular, since they are full objects, make the config.json much larger _and_ it is completely unnecessary as the aggregator doesnt need the full manifest in the json; it loads it from the configmap anyways.